charts/fleet-telemetry/README.md

```diff
@@ -57,3 +58,41 @@ tlsSecret:
     value of the cert PEM
   tlsKey: |
     value the private key
+config:
+  data: |
```

Comment:
can we optionally reference a file?

Reply:
no, helm chart doesn't support that, client can shell script to do it